kornia/augmentation/_2d/mix/mosaic.py

```diff
@@ -0,0 +1,179 @@
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import torch
+from torch.nn.functional import interpolate
+
+from kornia.augmentation import random_generator as rg
+from kornia.augmentation._2d.mix.base import MixAugmentationBaseV2
+from kornia.constants import DataKey, Resample
+from kornia.core import Tensor
+from kornia.geometry.boxes import Boxes
+from kornia.geometry.transform import crop_by_indices, crop_by_transform_mat, get_perspective_transform
+from kornia.utils import eye_like
+
+
+class RandomMosaic(MixAugmentationBaseV2):
+    r"""RandomMosaic.
+
+    Args:
+        min_bbox_size: minimum area of bounding boxes. Default to 0.
+        p: probability for applying an augmentation. This param controls if to apply the augmentation for the batch.
+        p_batch: probability for applying an augmentation to a batch. This param controls the augmentation
+          probabilities batch-wise.
+        same_on_batch: apply the same transformation across the batch.
+        keepdim: whether to keep the output shape the same as input ``True`` or broadcast it
+          to the batch form ``False``.
+
+    Examples:
+        >>> mosaic = RandomMosaic()
+        >>> input = torch.randn(8, 3, 16, 16)
+        >>> mosaic(input).shape
+    """
+
+    def __init__(
+        self,
+        output_size: Optional[Tuple[int, int]] = None,
```

Comment:
i like parameters that contains None in the last terms. Possibly the last before `data_keys` or `p`